tests/benchmarkstt/test_metrics_core.py

```diff
@@ -39,3 +39,29 @@ def test_wer(a, b, exp):
     assert WER(mode=WER.MODE_STRICT).compare(PlainText(a), PlainText(b)) == wer_strict
     assert WER(mode=WER.MODE_HUNT).compare(PlainText(a), PlainText(b)) == wer_hunt
     assert WER(mode=WER.MODE_LEVENSHTEIN).compare(PlainText(a), PlainText(b)) == wer_levenshtein
+
+
+@pytest.mark.parametrize('a,b,entities_list,weights, exp', [
+    ['madam is here', 'adam is here', ['madam', 'here'], [100, 10], (0.455, 2)],
+    ['madam is here', 'adam is here', ['madam', 'here'], [0.9, 0.1], (0.450, 2)],
+    ['madam is here', 'adam is here', ['madam', 'here'], [10, 100], (0.045, 2)],
+    ['theresa may is here', 'theresa may is there', ['theresa may', 'here'], [10, 100], (0.455, 2)],
+    ['theresa may is here', 'theresa may is there', ['theresa may', 'here'], [100, 10], (0.045, 2)],
+    ['aa bb cc dd', 'aa bb cc dd', ['aa', 'bb'], [100, 10], (0.0, 2)],
+    ['aa bb cc dd', 'aa bb cc dd', ['aa', 'bb'], [10, 100], (0.0, 2)],
+    ['aa bb cc dd', 'aa bb cc dd ee ff gg hh', ['aa', 'ee'], [10, 100], (0.0, 1)],
```

Comment:
Perhaps I don't understand the algorithm, but I think this example should have a BEER of >0. One string contains `ee` but the other doesn't.

Reply:
The entities 'aa' and 'bb' are both one time in the reference and in the hypothesis so the BEER is 0 and the number of occurrences is 2.